### PR TITLE
Testing: Math Magician

### PR DESCRIPTION
**Pull request by RitobrotoMukherjee**
### What we use 👨‍💻  
- [x]  React Testing Library.
- [x] snapshots with Jest.

### What we implement ♻️ 
- Set up React Testing Library
- Write unit tests for the files `operate.js` and `calculate.js` using Jest.
- Create unit tests for all React components
- We Run a full test suite using Jest.

## Technology used 🧑‍💻 
- Language : JavaScript(ES6), JSX and CSS3
- Framework: None
- Library: `Reactjs` , `react-testing-library` and `jest`
- Module bundler  Webpack, Babel
- Technology: Gitflow, Github Workflow, Linters

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,6 +74,26 @@ To get a local copy up and running follow these simple example steps.
 ![Chris Siku's GitHub stats](https://github-readme-stats.vercel.app/api?username=Chrissiku&count_private=true&theme=dark&show_icons=true)
```
Comment (zieeco) [inline README.md]: - Please, kindly remove this extra added content following the microverse [readme template](https://github.com/microverseinc/readme-template) to be consistent. This part could be added to the overview of your GitHub account.

File: README.md [hunk 2]
```diff
@@ -74,6 +74,26 @@ To get a local copy up and running follow these simple example steps.
...
 
+👤 **Ritobroto Mukherjee**
+
+- Portfolio: [@Rito](https://ritobrotomukherjee.github.io/Work-Portfolio/)
+- GitHub: [@RitobrotoMukherjee](https://github.com/RitobrotoMukherjee)
+- Twitter: [@RitobrotoM3](https://twitter.com/RitobrotoM3)
+- LinkedIn: [Ritobroto Mukherjee](https://www.linkedin.com/in/ritobroto-mukherjee-519148ba/)
+
+![Ritobroto's GitHub stats](https://github-readme-stats.vercel.app/api?username=RitobrotoMukherjee&count_private=true&theme=dark&show_icons=true)
+
```
Comment (zieeco) [inline README.md]: - Please, kindly remove this extra added content following the microverse [readme template](https://github.com/microverseinc/readme-template) to be consistent. This part could be added to the overview of your GitHub account.

File: README.md [hunk 3]
```diff
@@ -74,6 +74,26 @@ To get a local copy up and running follow these simple example steps.
...
+👤 **Kiprotich Kimutai**
+
+- GitHub: [@Kimutai01](https://github.com/Kimutai01)
+- Twitter: [@Kimutai_01](https://twitter.com/Kimutai_01?s=09)
+- LinkedIn: [@kiprotichkimutai](https://www.linkedin.com/m/in/kimutai-kiprotich-1b5045216)
+
+
+![Kimutai's GitHub stats](https://github-readme-stats.vercel.app/api?username=Kimutai01&count_private=true&theme=dark&show_icons=true)
+
+
```
Comment (zieeco) [inline README.md]: - Please, kindly remove this extra added content following the microverse [readme template](https://github.com/microverseinc/readme-template) to be consistent. This part could be added to the overview of your GitHub account.

File: README.md (hunk 2 above)
Comment (zieeco) [reply README.md]: - [x] Update your readme and add some instruction on how the user can run the test.

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @Team,


Great job so far! 👏 
You've done well implementing the test, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️
- Test are passing :100: 


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (Chrissiku) [reply README.md]: Thank you @zieeco for this advice

**Review by sja-thedude — APPROVED** [review]
## Project Approved 👍 

Hi Team, This is @sja-thedude your code reviewer.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

giphy

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Chrissiku** [comment]
Thank you @sja-thedude for the approval of this project

**Pull request merged**